pyphare/pyphare/core/gridlayout.py

```diff
@@ -288,8 +295,13 @@ def yeeCoordsFor(self, qty, direction, withGhosts=True):
         if qty in yee_centering_lower[direction] and qty not in yee_centering[direction]:
             qty = qty[0].upper() + qty[1:]
 
-        return yeeCoordsFor(self.origin, self.nbrGhosts(self.interp_order, yee_centering[direction][qty]),
-                            self.dl, self.box.shape, qty, direction, withGhosts=withGhosts)
+        if "centering" in kwargs:
+            centering = kwargs["centering"]
+        else:
+            centering= yee_centering[direction][qty]
```

Comment:
same here for `centering = kwargs.get(...)`

Reply:
believe it or not that's what I did first, but that didn't work and raised key error because `qty` is not in the dict...